### PR TITLE
BUG: ISSUE WITH DAILY ROSTER PROJECTION VIEW

### PR DESCRIPTION
**Pull request by NoBoneZ**
## Is this a Feature, Chore or Bug?
- [] Feature
- [] Chore
- [X] Bug

## Clearly and concisely describe the feature, chore or bug.
There was an undefined method causing the algorithim to break

## Analysis and design (optional)
Analyse and attach the design documentation

## Solution description
Describe your code changes in detail for reviewers.
The method was removed from frappe, i checked for its replacement, but could not find any, i added it to one_fm utils

## Is there a business logic within a doctype?
    - [] Yes
    - x[] No

## Output screenshots (optional)
Post the output screenshots, if a UI is affected or added due to this feature.

## Areas affected and ensured
List out the areas affected by your code changes.
Daily Roster Projection VIew

## Is there any existing behavior change of other features due to this code change?
Mention Yes or No. If Yes, provide the appropriate explanation.

## Did you test with the following dataset?
- [x] Existing Data
- [] New Data

## Was child table created?
    - [] is attachment required?
        did you test attachment
## Did you delete custom field?
    - [] Yes
    - [x] No
        If yes, did you write a delete patch?

## Is patch required?
- [] Yes
- [x] No
    ## Was the patch test?

## Which browser(s) did you use for testing?
  - [x] Chrome
  - [] Safari
  - [] Firefox

### CODE REVIEW

File: one_fm/tasks/one_fm/daily.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 from frappe.utils.data import get_url_to_form
 from frappe.desk.query_report import *
 from one_fm.processor import sendemail
+from one_fm.utils import background_enqueue_run
```
Comment (pjamsheer) [inline one_fm/tasks/one_fm/daily.py:8]: Why this import here

File: one_fm/utils.py [hunk 2]
```diff
@@ -3644,4 +3645,28 @@ def send_work_anniversary_reminders():
                     others = [d for d in anniversary_persons if d != person]
                     reminder_text = get_work_anniversary_reminder_text(others)
                     send_work_anniversary_reminder(person_email, reminder_text, others, message, sender)
+
+
+def background_enqueue_run(report_name, filters=None, user=None):
```
Comment (pjamsheer) [inline one_fm/utils.py:3650]: Where this method is using

File: one_fm/tasks/one_fm/daily.py (hunk 1 above)
Comment (NoBoneZ) [reply one_fm/tasks/one_fm/daily.py:8]: The method was a frappe method, but it was removed, and there was no replacement
@pjamsheer

File: one_fm/utils.py (hunk 2 above)
Comment (NoBoneZ) [reply one_fm/utils.py:3650]: This method is the one that was removed by frappe

**Review by pjamsheer — APPROVED** [review]

**Pull request merged**